src/components/VisualizationPlugin/__tests__/VisualizationPlugin.spec.js

```diff
@@ -167,15 +184,19 @@ describe('VisualizationPlugin', () => {
         })
 
         it('calls onError callback when an exception is thrown', async () => {
+            consoleErrorSuppressor.suppress()
             // eslint-disable-next-line no-import-assign, import/namespace
             api.apiFetchAnalytics = jest.fn().mockRejectedValue('error')
 
             await canvas()
 
+            consoleErrorSuppressor.restore()
+
             expect(defaultProps.onError).toHaveBeenCalled()
         })
```

Comment:
I guess that here you could argue that it's OK to suppress the console error. There are more elegant ways to assert errors in Jest, like [this one](https://jestjs.io/docs/expect#rejects), but I wouldn't know how you could then also assert that `defaultProps.onError` has been called.....